src/Command/AssertBackwardsCompatible.php

```diff
@@ -213,4 +220,24 @@ private function determineFromRevisionFromRepository(
             $repository,
         );
     }
+
+    private function determineConfiguration(
```

Comment:
I'd advocate for separating this into a separate invokable method in a class; I can imagine this will grow in complexity if things like different filename support, validating config schema etc. get added, so isolating that initially _may_ be a nice idea?

Reply:
Sure thing, having it extracted now would certainly ease further extensions.

Reply:
Alright, I had a few minutes today to play more with this and I've refactored things a bit on c251f5a to address this point (not yet implementing schema and all).

I've decided to keep the configuration determination logic in `src/Command` as, even though it isn't a command, it has dependencies to command stuff (`OutputInterface`).

Do you think it's sufficient for now? If so, I'll just replace the parsing/validation logic and we can call this a feature :smile: